kolibri/plugins/learn/assets/src/views/index.vue

```diff
@@ -101,7 +103,6 @@
       'breadcrumbs': require('./breadcrumbs'),
       'search-page': require('./search-page'),
       'tabs': require('kolibri.coreVue.components.tabs'),
-      'tab-link': require('kolibri.coreVue.components.tabLink'),
```

Comment:
I like how this change allows the whole `tabs` component to be used with a single `require`, and the child components simply "come along for the ride".